app/javascript/packs/application.js

```diff
@@ -45,6 +48,9 @@ Vue.component('IngredientsContainer', IngredientsContainer);
 Vue.component('RecipesContainer', RecipesContainer);
 Vue.component('RecipeShow', RecipeShow);
 Vue.component('CreateRecipes', CreateRecipes);
+Vue.component('addButtonComponent', addButton);
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name "addButtonComponent" is not PascalCase  vue/component-definition-name-casing